### PR TITLE
Add desktop alternative application

### LINKED ISSUE #787: Create an alternative desktop app
Create an alternative app to test and demonstrate the alternative comportment like:

* Theme (add)
* Query in a table
* Non grouped search

### PR DESCRIPTION
**Pull request by sbrunner**
fix #787

result: http://camptocamp.github.io/ngeo/desktop_alt/examples/contribs/gmf/apps/desktop_alt/index.html

### CODE REVIEW

File: contribs/gmf/apps/desktop_alt/js/controller.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+goog.require('app');
+goog.require('gmf.AbstractDesktopController');
+/** @suppress {extraRequire} */
+goog.require('gmf.authenticationDirective');
+/** @suppress {extraRequire} */
+goog.require('ngeo.proj.EPSG21781');
+
+
+app.module.constant(
```
Comment (fredj) [inline contribs/gmf/apps/desktop_alt/js/controller.js]: Not needed, already in index.html

Comment (sbrunner) [inline contribs/gmf/apps/desktop_alt/js/controller.js]: Thanks, fixed :-)

File: contribs/gmf/apps/desktop/js/controller.js [hunk 2]
```diff
@@ -19,11 +19,6 @@ goog.require('gmf.authenticationDirective');
 goog.require('ngeo.proj.EPSG21781');
 
 
-app.module.constant(
```
Comment (ger-benjamin) [inline contribs/gmf/apps/desktop/js/controller.js]: Why this removal ? Just one uselesss thing that you has seeing (but no relation with this PR) ?

**Comment by ger-benjamin** [comment]
LGTM

File: contribs/gmf/apps/desktop/js/controller.js (hunk 2 above)
Comment (sbrunner) [inline contribs/gmf/apps/desktop/js/controller.js]: As the commit say duplicate code :-)

**Pull request merged**